src/madhavi_Khasbage/SwitchCaseNested/BankManagement.java

```diff
@@ -0,0 +1,137 @@
+package madhavi_Khasbage.SwitchCaseNested;
+import java.util.Scanner;
+
+public class BankManagement {
+	double iciciBalance = 1000, hdfcBalance = 2000, bobBalance = 3000, sbiBalance = 4000;
+	int idebitCount=0, icreditCount=0,hdebitCount=0,hcreditCount=0,bdebitCount=0,bcreditCount=0,sdebitCount=0,screditCount=0;
+	
+
+	void banksAccount(String bankName, String accType, double amount) {
+		double remainBalance;
+		accType = accType.toLowerCase();
+		bankName = bankName.toLowerCase();
+
+		switch (accType) {
+		/* debit case start*/
+		case "debit":			
+			switch (bankName) {
+			case "icici":
+				if (amount <= iciciBalance) {
```

Comment:
instead of writing multi-time same code block you can write debit() method which accept the two arguments and you can simply call that method